### PR TITLE
`pip install pyjs`

### PR DESCRIPTION
**Pull request by duly**
https://github.com/pypa/pip/issues/1649
https://github.com/pyjs/pyjs/issues/742

Evidence follows that the patch works in python 3.3.2 and it should also be python 2.7 compatible.
setup.py previously was not compatible with python 3.3.2.

https://github.com/pyjs/pyjs/issues/742#issuecomment-37746937

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -14,4 +26,9 @@
     package_data=package_data,
     zip_safe = False,
     entry_points = entry_points,
-    )
+    
+    license="Apache",
+    keywords="js",
```
Comment (glyph) [inline setup.py]: Technically a single string is valid here, but I think a list of strings would make it clearer how to add additional keywords later.  So: `keywords=["js", "javascript"]`.

**Comment by glyph** [comment]
This looks basically good to me, but I'm not in a position to merge code on this project :-).

**Comment by duly** [comment]
Thank you Glyph for the feedback and suggestion.

File: setup.py (hunk 1 above)
Comment (duly) [inline setup.py]: Per @glyph 's suggestion to our setup.py. We added additional keywords. We are unaware of cross-version python support at this time. https://github.com/duly/pyjs/commit/5c0082b9fb62edad8880c9d546acb77009612f67

**Pull request merged**